src/containers/Permissions/index.jsx

```diff
@@ -477,7 +478,10 @@ class Permissions extends React.Component {
 	}
 
 	render() {
+		const { showLoader } = this.props;
```

Comment:
when we change route and get back to permission page, we could see that load is off
one of ways to solve that, when clear for Permission page do not clear loader status.